core/src/main/java/org/hibernate/ogm/jpa/impl/OgmEntityManager.java

```diff
@@ -633,39 +616,4 @@ public CriteriaBuilder getCriteriaBuilder() {
 	public Metamodel getMetamodel() {
 		return hibernateEm.getMetamodel();
 	}
-
-	/**
```

Comment:
Restored it.

Noticed though that there is a related issue: `o.h.jpa.internal.TransactionImpl` triggers another conversion via `org.hibernate.jpa.spi.HibernateEntityManagerImplementor#convert()`. There we can't intercept as this is ORM's own EM impl. I think we can handle it separately, I'll open an issue.